src/lib/y2partitioner/device_graphs.rb

```diff
@@ -0,0 +1,47 @@
+require "singleton"
+
+module Y2Partitioner
+  # Object that holds and manipulates of storage device graphs.
+  class DeviceGraphs
```

Comment:
in general I would like to skip in this PR discussion about transaction, as I just cherry pick this part from other pull request and I do not use it at all.
About simplifying access to class it is fine for me.